Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/TeleOpComp.java

```diff
@@ -157,51 +153,62 @@ public void runOpMode() {
             if (gamepad2.y){
                 position = -1;
             }
+            else if (gamepad2.b){
+                position = -2;
+            }
 
-            if (gamepad1.right_bumper) {
+            if (gamepad2.right_bumper) {
+                motorBR.setPower(0);
+                motorBL.setPower(0);
+                motorFL.setPower(0);
+                motorFR.setPower(0);
                 x = 0.7;
                 while (true) {
```

Comment:
It's generally bad practice to use a while true loop. Try to avoid it if you can.